### DISCUSSION TITLE
Question about launching with android.intent

### DISCUSSION THREAD

**Discussion by stl3** [pystardust/ani-cli#1255] [Q&A] [answered]
```
am start --user 0 -a android.intent.action.VIEW -d "$episode" -n is.xyz.mpv/.MPVActivity 
am start --user 0 -a android.intent.action.VIEW -d "$episode" -n org.videolan.vlc/org.videolan.vlc.gui.video.VideoPlayerActivity
```

I've been wondering for a while but is there any way to pass extra parameters to mpv (or vlc) from the intent or is it not possible? I do see options like -e/--es etc but I don't think they might be applicable here. Just wanted to pass subtitles in termux. The only other way for me to load subtitles was to let it download to `/data/data/com.termux/files/home/storage/downloads`  (have to put it here to be 'visible' outside Termux) and manually load the subtitle file when mpv or vlc launches, other than having the title of the sub match the video file title in the same dir (mpv could probably do this through mpv.conf or script too but that's kind of the long undesirable way). Any ideas or is it a no go?

**Comment by CoolnsX** [comment, marked as answer]
I am trying to make subtitles work with mpv, but no luck. Mpv has subtitles passing intent but I have been unsuccessful so far
